config/operators/pipelines/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator
+  namespace: openshift-operators
+spec:
+  channel: canary
+  installPlanApproval: Automatic
+  name: openshift-pipelines-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: openshift-pipelines-operator.v0.11.0
```

Comment:
we cannot use the latest OSP until we migrate all our pipelines to the `v1beta1`

Reply:
Right. Let's stick with 0.10.7 for now - https://github.com/codeready-toolchain/toolchain-infra/pull/12/commits/20dcebea5e0ef5c72e5def50c139550c936b1afd

Reply:
May I ask you for moving it back to v0.11.0? 
I made the OSP operator running for this latest version in our member cluster, so I'm working on migrating our pipelines to the `v1beta1` api

Reply:
Done in https://github.com/codeready-toolchain/toolchain-infra/pull/12/commits/aac812e4b608ec84aaeb0343b662c7f597b97cc8